pandas/core/util/numba_.py

```diff
@@ -83,6 +86,12 @@ def jit_user_function(func: Callable) -> Callable:
     if numba.extending.is_jitted(func):
         # Don't jit a user passed jitted function
         numba_func = func
+    elif getattr(np, func.__name__, False) is func or isinstance(
+        func, types.BuiltinFunctionType
+    ):
+        # Not necessary to jit builtins or np functions
+        # This will mess up register_jitable
+        numba_func = func
```

Comment:
Sorry for the slow reply here.

I think this should be fine - the return type in the docstring says it can return a function that is JITable by numba.

(In the pandas code, I also don't think there's anywhere where we directly apply the jitted user function on data (we always pass it to another jitted function that calls the output from jit_user_function).)

Will add a test for this, though.

Reply:
Thanks for checking this and adding the test.